ethcore/light/src/client/header_chain.rs

```diff
@@ -53,6 +59,9 @@ const HISTORY: u64 = 2048;
 /// The best block key. Maps to an RLP list: [best_era, last_era]
 const CURRENT_KEY: &'static [u8] = &*b"best_and_latest";
 
+/// Key storing the last canonical epoch transition.
+const LAST_CANONICAL_TRANSITION: &'static [u8] = &*b"canonical_transition";
```

Comment:
Does this require a database upgrade?

Reply:
No, the most recent version of the code instantiates this key if it doesn't exist already. Only chains which don't use epochs can be synced by the light client right now anyway. When running the new version, their genesis epoch data will be populated correctly.